WallsAndHoles/editor.cpp

```diff
@@ -100,12 +100,11 @@ Editor::~Editor()
 void Editor::newMap()
 {
     NewMapDialog nmd;
-    nmd.exec();
```

Comment:
QDialog::exec() returns a value based on the input of the dialog. (Basically 1 for accept and 0 for reject)